frontend/src/components/Admin/CoffeeChats/CoffeeChats.tsx

```diff
@@ -30,39 +40,104 @@ const CoffeeChats: React.FC = () => {
-                    <div className={styles.pending_text}>
-                      {chatCount.get(category) ? `${chatCount.get(category)} pending` : ''}{' '}
-                    </div>
+    <div className={styles.flexContainer}>
+      <div>
+        {selectedMember ? (
+          <div>
+            <div className={[styles.formWrapper, styles.wrapper].join(' ')}>
+              <h1>
+                Bingo Board for {selectedMember.firstName} {selectedMember.lastName}
+              </h1>
+            </div>
+            <div className={styles.bingoBoardWrapper}>
+              <CoffeeChatsBingoBoard
+                approvedChats={specificApprovedChats}
+                pendingChats={specificPendingChats}
+                rejectedChats={specificRejectedChats}
+                isChatLoading={isChatLoading}
+                bingoBoard={bingoBoard}
+              />
+            </div>
+          </div>
+        ) : (
+          <div>
+            <div className={[styles.formWrapper, styles.wrapper].join(' ')}>
+              <h1>Review Coffee Chats</h1>
+            </div>
+            <div className={styles.wrapper}>
+              <div className={styles.bingo_board}>
+                {bingoBoard.flat().map((category, index) => (
+                  <div key={index}>
+                    <Link
+                      key={category}
+                      href={{
+                        pathname: `/admin/coffee-chat-details/${hashString(category)}`,
+                        query: { category }
+                      }}
+                    >
+                      <div className={styles.bingo_cell}>
+                        <div style={{ display: 'flex', flexDirection: 'column' }}>
+                          <div className={styles.bingo_text}>{category}</div>
+                          <div className={styles.pending_text}>
+                            {chatCount.get(category) ? `${chatCount.get(category)} pending` : ''}{' '}
+                          </div>
+                        </div>
+                      </div>
+                    </Link>
                   </div>
-                </div>
-              </Link>
+                ))}
+              </div>
             </div>
-          ))}
+          </div>
+        )}
+        <div className={styles.buttonContainer}>
+          <Link href="/admin/coffee-chats/dashboard">
+            <Button>View Coffee Chats Dashboard</Button>
+          </Link>
         </div>
       </div>
-      <div className={styles.buttonContainer}>
-        <Link href="/admin/coffee-chats/dashboard">
-          <Button>View Coffee Chats Dashboard</Button>
-        </Link>
+      <div>
+        <div className={styles.dropdownContainer}>
+          <Button onClick={() => setSelectedMember(null)} disabled={selectedMember == null}>
+            Review All Coffee Chats
+          </Button>
+          <div className={styles.dropdownButton}>
+            <Dropdown
+              placeholder={'View Member Bingo Board'}
```

Comment:
Can you make this dropdown searchable? You can add the `search` prop to the `Dropdown` component to do this.

Also, could you show the NetID's alongside the people's names (just in case there's a duplicate first name/last name combination), and make sure the dropdown is searchable by first name, last name, and netid?